.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Changesets
+on:
+  push:
+    branches:
+      - main
+      - canary
```

Comment:
If we're only publishing based on main, should canary be removed from the list here?

Reply:
True, good catch!